src/js/main.js

```diff
@@ -4,10 +4,14 @@ import Section from './components/Section.vue';
 import ItemLink from './components/ItemLink.vue';
 import ItemBirthday from './components/ItemBirthday.vue';
 import VModal from 'vue-js-modal';
+import IconSelect from './components/IconSelect.vue';
+import VSelect from 'vue-select';
 
 Vue.component('sections', Section);
 Vue.component('item-link', ItemLink);
 Vue.component('item-birthday', ItemBirthday);
+Vue.component('icon-select', IconSelect);
+Vue.component('v-select', VSelect);
```

Comment:
Another thing: if you are using this component only on `IconSelect` component, you can import it directly there.